test/translator/read_test.cc

```diff
@@ -100,6 +107,7 @@ TEST_F(ReadTest, Read6ToNvmeShouldRead256BlocksForZeroTransferLen) {
   uint8_t network_endian_lba_1 = 0x1a;
   uint16_t network_endian_lba_2 = htons(0x2b3c);
   uint32_t cdw10 = 0x1a2b3c;
+  uint16_t transfer_len = 256;
```

Comment:
Can you add a comment here that technically the transfer_length is set to zero when building the SCSI command but it "means" 256